docs/docs/docs/getting-started/installation.md

```
@@ -291,29 +291,29 @@ These steps are specific to Linux. You will need to modify them accordingly for
-{"outcome":"success","imageName":"talawa-api"}
-```
+   ```
+   {"outcome":"success","imageName":"talawa-api"}
+   ```
 
 1. Start the docker devcontainer
 
-```
-devcontainer up --workspace-folder .
-```
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)